### PR TITLE
Update tradfri installation instructions

### PR DESCRIPTION
**Pull request by lwis**
**Description:**

Update instructions for aiocoap installation.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7815

### CODE REVIEW

**Comment by Danielhiversen** [comment]
Should this be against the current branch?

**Comment by lwis** [comment]
I wasn't sure if we usually cherry pick or back merge? I can switch if necessary.

File: source/_components/tradfri.markdown [hunk 1]
```diff
@@ -14,44 +14,42 @@ ha_iot_class: "Local Polling"
...
-$ brew install automake
-$ git clone --depth 1 --recursive -b dtls https://github.com/home-assistant/libcoap.git
-$ cd libcoap
-$ ./autogen.sh
-$ ./configure --disable-documentation --disable-shared --without-debug CFLAGS="-D COAP_DEBUG_FD=stderr"
-$ make
-$ make install
+python3 -m pip install cython
+
+cd /usr/src/app/
```
Comment (leppa) [inline source/_components/tradfri.markdown]: Not everybody has `/usr/src/app`. This script will fail for them. I'd either `mkdir` it or use `tmp`. The source is not needed after the installation, anyway.

Comment (lwis) [reply source/_components/tradfri.markdown]: Good spot.

File: source/_components/tradfri.markdown [hunk 2]
```diff
@@ -14,44 +14,41 @@ ha_iot_class: "Local Polling"
...
-$ brew install libtool
-$ brew install autoconf
-$ brew install automake
-$ git clone --depth 1 --recursive -b dtls https://github.com/home-assistant/libcoap.git
-$ cd libcoap
-$ ./autogen.sh
-$ ./configure --disable-documentation --disable-shared --without-debug CFLAGS="-D COAP_DEBUG_FD=stderr"
-$ make
-$ make install
+python3 -m pip install cython
```
Comment (fabaff) [inline source/_components/tradfri.markdown]: This should be done inside the venv if there is any.

File: source/_components/tradfri.markdown [hunk 3]
```diff
@@ -14,44 +14,41 @@ ha_iot_class: "Local Polling"
...
+
+mkdir build && cd build
+
+git clone --depth 1 https://git.fslab.de/jkonra2m/tinydtls
+cd tinydtls
+autoreconf
```
Comment (fabaff) [inline source/_components/tradfri.markdown]: This requires that `autoconf` is available.

Comment (fabaff) [reply source/_components/tradfri.markdown]: On Fedora, and I guess on CentOS as well, `automake` required too.

**Pull request merged**